core/src/main/java/cucumber/runtime/RuntimeOptions.java

```diff
@@ -114,11 +115,11 @@ private void parse(List<String> args) {
                 parsedGlue.add(gluePath);
             } else if (arg.equals("--tags") || arg.equals("-t")) {
                 parsedFilters.add(args.remove(0));
-            } else if (arg.equals("--plugin") || arg.equals("-p")) {
-                addPluginName(args.remove(0));
+            } else if (arg.equals("--plugin") || arg.equals("--add-plugin") || arg.equals("-p")) {
```

Comment:
Oh wait, you're changing the semantics of `--plugin` entirely! Hm. Same thought process applies. Since it's already released, should we use `--add-plugin` and have those semantics be different than the existing `--plugin`?